src/test/daml/Daml/Finance/Settlement/Test/Util.daml

```diff
@@ -0,0 +1,178 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Settlement.Test.Util where
+
+import DA.Action (foldlA)
+import DA.List (head)
+import DA.Set qualified as S (fromList)
+import Daml.Finance.Account.Account qualified as Account (T)
+import Daml.Finance.Interface.Account.Account qualified as Account (I, toKey)
+import Daml.Finance.Interface.Holding.Base qualified as Holding (I)
+import Daml.Finance.Interface.Settlement.Batch qualified as Batch (Cancel(..), I, Settle(..))
+import Daml.Finance.Interface.Settlement.Instruction qualified as Instruction (Allocate(..), Approve(..), I, toKey)
+import Daml.Finance.Interface.Settlement.Types (Allocation(..), Approval(..), InstructionKey(..))
+import Daml.Finance.Interface.Types.Common.Types (Id(..))
+import Daml.Finance.Interface.Util.SyntheticKey  (IndexTable(..), insert)
+import Daml.Finance.Settlement.Instruction qualified as Instruction (T)
+import Daml.Finance.Util.Common (mapWithIndex)
+import Daml.Script
+
+-- | Utility functions for allocating an instruction.
+allocateAny : [Party] -> ContractId Instruction.I -> Allocation ->
+  Script ((ContractId Instruction.I, Optional (ContractId Holding.I)), IndexTable)
```

Comment:
```suggestion
  Script (ContractId Instruction.I, Optional (ContractId Holding.I))
```

I would not return the `IndexTable` as it seems to never be used in the tests

Reply:
agree